apps/logic-apps-ux-docs/docs/tsg.mdx

```diff
@@ -0,0 +1,91 @@
+
+[![Known Issues Image](./img/tsg/knownIssuesExample_light.png#light-mode-only)](https://github.com/Azure/LogicAppsUX/issues/2402)
+
+[![Known Issues Image](./img/tsg/knownIssuesExample_dark.png#dark-mode-only)](https://github.com/Azure/LogicAppsUX/issues/2402)
+
+If you determine your issue is unique, please **[submit a new issue](https://github.com/Azure/LogicAppsUX/issues/new?assignees=&labels=&projects=&template=bug_report.yml)** and start collecting the information requested in the issue template.
+
+:::note Note on releases
+
+We release new versions of Logic Apps Designer every 2 weeks. Depending on where we are in our current cycle when you submit your issue, it may take up to 2 weeks for us to release a fix for your issue after it is complete. We will do our best to mitigate your issue as soon as possible, and if the issue is severe enough we will potentially release a hotfix.
+
+:::
+
+### Providing your workflow
+
+If you are experiencing issues with a specific workflow, please provide the workflow definition if you are able. We understand that some workflows may contain sensitive information, so please remove any sensitive information before providing the workflow definition. If you are unable to provide the workflow definition, please provide a description or screenshots of the workflow and any important actions or connectors you think may be causing the issue.
+
+### Collecting console errors
+
+Providing console errors are very helpful for our engineers to quickly pinpoint the source of an issue. To collect console errors, please follow the steps below:
+
+1. Open the developer tools in your browser with the keyboard shortcut `Ctrl+Shift+I`
+2. Select the **Console** tab
+3. If you see any errors, you can right-click on the error and select **Save as...** to save the error to a text file
+4. Otherwise you can also copy/paste as needed
+
+### Collecting network logs in a HAR file
+
+If you are experiencing issues with the designer loading, or with any network requests, collecting a HAR file will help us investigate the issue. To collect a HAR file, please follow the steps below:
+
+1. Open the developer tools in your browser with the keyboard shortcut `Ctrl+Shift+I`
+2. Select the **Network** tab
+3. Refresh the page and reproduce the issue
+4. Once you have reproduced the issue, right-click anywhere in the grid of network requests, select **Save as HAR with Content**, and save the file to your computer
+5. Your HAR file is now ready to be sent
+
+:::caution Warning on sensitive information
+
+These network logs may contain sensitive information, so please do not share them within GitHub issues. If an engineer requests them, please send them via email. (Should be supplied when requesting the HAR)
```

Comment:
We should also mention about opening tickets with CSS or incident if this is for CSS and sharing them there, depending on the severity of the issue